src/App.js

```diff
@@ -1,16 +1,21 @@
+/* eslint-disable */
 import React from 'react';
-import Calculator from './Components/Calculator';
-import './App.css';
+import { Outlet } from 'react-router-dom';
+import Navigation from './components/Navigation';
+import Footer from './components/Footer';
```

Comment:
- [ ] At the moment, the app in unable to start because these modules are imported wrongly. Please kindly correct the path so they can be properly imported. For example:
`import Navigation from './Components/Navigation`